subsystems/DrivetrainBase.java

```diff
@@ -65,8 +126,8 @@ public void tankDrive(double leftThrottle, double rightThrottle) {
 
     // a tank drive that sets the voltages of the motors
     public void tankDriveVolts(double leftVolts, double rightVolts) {
-        leftGroup.setVoltage(-leftVolts);
-        rightGroup.setVoltage(rightVolts);
+        leftGroup.setVoltage(invertLeft * leftVolts);
```

Comment:
leave inverting up to the `configureMotors` method that will be in the implementing class